src/taipy/config/config.pyi

```diff
@@ -89,15 +82,20 @@ class Config:
         Parameters:
             filename (Union[str, Path]): The path of the toml configuration file to load.
         """
-
     @classmethod
     def block_update(cls):
         """Block update on the configuration signgleton."""
-
     @classmethod
     def unblock_update(cls):
         """Unblock update on the configuration signgleton."""
-
+    @classmethod
+    @_ConfigBlocker._check()
+    def _register_default(cls, default_section: Section):
+        """"""
+    @classmethod
+    @_ConfigBlocker._check()
+    def _register(cls, section):
+        """"""
```

Comment:
The methods are hidden. DO we need them in the pyi file ?

Reply:
It is to resolve a mypy error message on __init__.py, `_inject_section()` method.

Reply:
Oh right it is used outside taipy-config.

Thx!